man/virt-install.rst

```diff
@@ -897,10 +897,12 @@ Sub options are:
 ``meta-data=``
     Specify a cloud-init meta-data file to add directly to the iso. All other
     meta-data configuration options on the --cloud-init command line are ignored.
+    Can by path to local file or URL.
```

Comment:
`Can by` -> `Can be`, same with other files

Reply:
Will fix that, thanks.